native/cocos/bindings/manual/jsb_module_register.cpp

```diff
@@ -167,6 +171,10 @@ bool jsb_register_all_modules() {
     se->addRegisterCallback(register_script_native_bridge);
 #endif
 
+#if (CC_PLATFORM == CC_PLATFORM_OPENHARMONY)
+    se->addRegisterCallback(register_javascript_arkTs_bridge);
+#endif
```

Comment:
On openharmony, it may not use ark. Then is it ok to register it?

Reply:
The UI thread must be ark, it's just that our cocos engine is not necessarily running on ark